.travis.yml

```diff
@@ -104,6 +104,7 @@ matrix:
       - if [ "$TRAVIS_OS_NAME" == "linux" ]; then sudo ln -s -v -f $(which gcc-$GCC_VERSION) /usr/bin/gcc; fi
 
       script:
+      - pip install cmake
```

Comment:
why are the changes in this file needed? CI has always worked with the existing CMake.

Reply:
https://github.com/grpc/grpc/blob/master/CMakeLists.txt#L60-L66
https://github.com/grpc/grpc/blob/master/BUILDING.md#install-after-build 
Recent gRPC does not support installing with compiling third-party on cmake < 3.13. If you don't want to update the cmake, you should manually install some third-party libraries. `gRPC_*_PROVIDER=package` is the only supported option for the cmake < 3.13, But It requires external dependencies.